tests/emit-removed-ambient.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+gdbus emit --system --object-path '/xyz/openbmc_project/sensors'            \
+           --signal org.freedesktop.DBus.ObjectManager.InterfacesRemoved    \
+           'objectpath "/xyz/openbmc_project/sensors/temperature/ambient"'  \
```

Comment:
Ehm... what's the use case? When can be an ambient temperature sensor removed?

Reply:
this command maybe used  in debug process to emulate of removing any sensors. 
`ambient` was chooses randomly.